spec/utils.js

```diff
@@ -36,12 +36,13 @@ function cleanup() {
 }
 
 const defaultOptions = {
-    url: '/',
+    url: './',
```

Comment:
Neutralinojs local URLs typically begin with `/` -- we need to remove the '.' character from the start